docs/usage/registry-cache/monitoring.md

```diff
@@ -0,0 +1,72 @@
+
+These metrics can also be viewed in a dedicated dashboard named `Registry Caches` in the Shoot's Plutono instance. From the `Registry` dropdown menu you can select the upstream for which you wish the metrics to be displayed (by default, metrics are summed for all upstream registries).
+
+Following is a list of all exposed `registry` metrics. The `upstream_host` label can be used to determine the upstream host to which the metrics are related, while the `type` label can be used to determine weather the metric is for an image `blob` or an image `manifest`:
+
+#### registry_proxy_requests_total
+The number of total incoming `registry` request received.
+- Type: Counter
+- Labels: `upstream_host` `type` 
+
+#### registry_proxy_hits_total
+The number of total cache hits; i.e. the requested content exists in the `registry` image store.
+- Type: Counter
+- Labels: `upstream_host` `type`
+
+#### registry_proxy_misses_total
+The number of total cache misses; i.e. the requested content is fetched from the upstream.
+- Type: Counter
+- Labels: `upstream_host` `type`
+
+#### registry_proxy_pulled_bytes_total
+The size of total bytes that `registry` has pulled from the upstream.
+- Type: Counter
+- Labels: `upstream_host` `type`
+
+#### registry_proxy_pushed_bytes_total
+The size of total bytes pushed to the `registry` clients.
+- Type: Counter
+- Labels: `upstream_host` `type`
+
+## Alerts
+
+There are two alerts defined for the `registry` cache `PersistentVolume` in the Shoot's Prometheus instance:
+
+#### RegistryCachePersistentVolumeUsageCritical
+This indicates that the `registry` cache `PersistentVolume` is almost full and less than 5% is free. When there is no available disk space, no new images will be cached. However, image pull operations are not affected. An alert is fired when the following expression evaluates to true:
+
+```
+100 * (
+ kubelet_volume_stats_available_bytes{persistentvolumeclaim=~"^cache-volume-registry-.+$"}
+   /
+ kubelet_volume_stats_capacity_bytes{persistentvolumeclaim=~"^cache-volume-registry-.+$"}
+) < 5
+```
+
+#### RegistryCachePersistentVolumeFullInFourDays
+This indicates that the `registry` cache `PersistentVolume` is expected to fill up within four days based on recent sampling. An alert is fired when the following expression evaluates to true:
+
+```
+100 * (
+ kubelet_volume_stats_available_bytes{persistentvolumeclaim=~"^cache-volume-registry-.+$"}
+   /
+ kubelet_volume_stats_capacity_bytes{persistentvolumeclaim=~"^cache-volume-registry-.+$"}
+) < 15
+and
+predict_linear(kubelet_volume_stats_available_bytes{persistentvolumeclaim=~"^cache-volume-registry-.+$"}[30m], 4 * 24 * 3600) <= 0
+```
+
+Users can subscribe to these alerts by following the Gardener [alerting guide](https://github.com/gardener/gardener/blob/master/docs/monitoring/alerting.md#alerting-for-users).
```

Comment:
Adapt https://github.com/gardener/gardener-extension-registry-cache/blob/0864c444f0c7ff028906e95d5dc65659f07ce9e4/docs/usage/registry-cache/configuration.md#L87 to link the alerting section in this guide.